test/onnx/gen_onnx.py

```diff
@@ -4117,7 +4117,7 @@ def scatter_test():
     y = helper.make_tensor_value_info('y', TensorProto.FLOAT, [3, 4, 5, 6])
 
     node = onnx.helper.make_node(
-        'Scatter',
```

Comment:
Can you generate more onnx files with different reduction attributes? Then you can add the tests under onnx_test.cpp.

Reply:
Then the *.pyc file should be in .gitignore, shouldn't it?

Reply:
yes, there is a PR with that change. But for now, you can manually remove it.

Reply:
done, commit b36bda7a96